pkg/cmd/openshift-kube-apiserver/openshiftkubeapiserver/patch_handlerchain.go

```diff
@@ -25,10 +25,14 @@ const (
 func BuildHandlerChain(genericConfig *genericapiserver.Config, kubeInformers informers.SharedInformerFactory, legacyServiceServingCertSignerCABundle string, oauthConfig *configapi.OAuthConfig, userAgentMatchingConfig configapi.UserAgentMatchingConfig) (func(apiHandler http.Handler, kc *genericapiserver.Config) http.Handler, map[string]genericapiserver.PostStartHookFunc, error) {
 	extraPostStartHooks := map[string]genericapiserver.PostStartHookFunc{}
 
-	webconsoleProxyHandler, err := newWebConsoleProxy(kubeInformers, legacyServiceServingCertSignerCABundle)
+	webconsoleProxyHandler, proxyPostStartHooks, err := newWebConsoleProxy(kubeInformers, legacyServiceServingCertSignerCABundle)
```

Comment:
Can you find a way to contain all this to the `hypershift openshift-kube-apiserver` command?  This new concept is only needed in `cluster up`.